lib/modules/platform/codecommit/index.md

```diff
@@ -142,3 +142,59 @@ module.exports = {
   ],
 };
 ```
+
+## CodeBuild examples
+
+Create a repository with `buildspec.yml`.
+This repository will be your BuildProject job repository to run Renovate on your repositories.
+
+### Renovate docker buildspec.yml 
+
+```yml
+version: 0.2
+env:
+  shell: bash
+  git-credential-helper: yes
+  variables:
+    RENOVATE_PLATFORM: 'codecommit'
+    RENOVATE_REPOSITORIES: '["repoName1", "repoName2"]'
+    RENOVATE_CONFIG: '{"extends":["config:base"]}'
+    LOG_LEVEL: 'debug'
+    AWS_REGION: 'us-east-1'
+phases:
+  build:
+    on-failure: CONTINUE
+    commands:
+      - docker run --rm -e AWS_REGION -e RENOVATE_CONFIG -e RENOVATE_PLATFORM -e RENOVATE_REPOSITORIES -e LOG_LEVEL renovate/renovate
+```
+
+### Renovate cli buildspec.yml 
+
+```yml
+version: 0.2
+env:
+  shell: bash
+  git-credential-helper: yes
+  variables:
+    RENOVATE_PLATFORM: 'codecommit'
+    RENOVATE_REPOSITORIES: '["repoName1", "repoName2"]'
+    RENOVATE_CONFIG: '{"extends":["config:base"]}'
+    LOG_LEVEL: 'debug'
+    AWS_REGION: 'us-east-1'
+phases:
+  build:
+    on-failure: CONTINUE
+    commands:
+      - npm install -g renovate
+      - renovate
+```
+
+### Note:
+
+In order to reduce Renovate BuildProject time and avoid `npm install`.
```

Comment:
I think we need some more explanation here.

Reply:
I prefer to merge this as it's a big improvement over what we have